sycl/doc/GetStartedWithSYCLCompiler.md

```diff
@@ -117,25 +117,57 @@ should be used.
 To run SYCL  applications on OpenCL devices, OpenCL implementation(s) must be
 present in the system.
 
-To execute SYCL tests on Intel devices following runtimes should be installed:
+Please, refer to [the Release Notes](../ReleaseNotes.md) for recommended Intel
+runtime versions.
 
-**Linux**
+The `GPU` runtime that is needed to run SYCL application on Intel `GPU` devices
+can be downloaded from the following web pages:
 
-* Intel GPU [Intel&reg; Graphics Compute Runtime for
+* Linux: [Intel&reg; Graphics Compute Runtime for
    OpenCL&trade;](https://github.com/intel/compute-runtime/releases)
-* Intel CPU - [Experimental Intel&reg; CPU Runtime for OpenCL&trade; Applications
-   with SYCL support](https://github.com/intel/llvm/releases)
-* Intel FPGA - TBD.
 
-**Windows**
-* OpenCL&trade; runtime for Intel GPU [Intel&reg; Download
+* Windows: [Intel&reg; Download
    Center](https://downloadcenter.intel.com/product/80939/Graphics-Drivers)
 
-* The experimental Intel&reg; CPU Runtime for OpenCL&trade; Applications with
-  SYCL support on Windows will be available soon.
 
-Please, refer to [the Release Notes](../ReleaseNotes.md) for recommended Intel
-runtime versions.
+To install Intel `CPU` runtime for OpenCL devices the corresponding runtime
+asset/archive should be downloaded from
+[SYCL Compiler and Runtime updates](../ReleaseNotes.md) and installed using
+the following procedure.
+
+**Linux**
+
+1) Extract the archive. For example, for archive with the name
+'oclcpu_rt_new_version.tar.gz' you would run the following commands
```

Comment:
I would use replace `oclcpu_rt_new_version` with `oclcpu_rt_<version>` everywhere.

Reply:
Ok, I have just force-pushed the requested fix. Thank you.